### PR TITLE
File structure

### PR DESCRIPTION
**Pull request by AymanAlqoqa**
relates #3 
file structure is created

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/FACG6/leannky-Anty/issues"
+  },
+  "homepage": "https://github.com/FACG6/leannky-Anty#readme",
+  "dependencies": {
+    "body-parser": "^1.18.3",
+    "cookie-parser": "^1.4.4",
+    "dotenv": "^7.0.0",
```
Comment (hshahwan) [inline package.json:24]: can you explain what 'dotenv' do?

Comment (AymanAlqoqa) [reply package.json:24]: it is used for environment variables, we need it in database connection with heroku database

**Review by naremanhilles — APPROVED** [review]

**Review by denaHS — APPROVED** [review]

**Review by alaabadra — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (AymanAlqoqa) [reply package.json:24]: please, merge the PR if there is no any issue.
thanks

Comment (hshahwan) [reply package.json:24]: it's new to me.
but we can deal with the DB variables without it dotenv.

**Pull request merged**